lib/pages/profile/bottom_sheet/hamburger_bts.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:clone_instagram/constants/SourceString.dart';
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+class HamburgerBts extends StatelessWidget{
+  const HamburgerBts({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    double width = MediaQuery.of(context).size.width;
```

Comment:
Move out of build function. Because if `setState` is called, it will be redefined (unnecessary).
Change it to getter (due to no setter need)